source/documentors/concepts/content_types.rst

```diff
@@ -1,49 +1,342 @@
-Documentation Content Types
-###########################
+Open edX Diataxis Guide
+#######################
 
-To keep documentation streamlined and make it easy for users to find answers, we make each topic one of four types:
+.. contents:: 
+  :local:
+  :depth: 1
+
+Diataxis Definition
+*******************
+
+The Diataxis framework is an approach to quality in technical documentation and creates a systematic organization. Diataxis identifies four modes of documentation:
```

Comment:
Recommend linking to Diataxis homepage (I know you do it in a Note at the end, but that's pretty far away from here):

```suggestion
The `Diataxis framework <https://diataxis.fr/>`_ is an approach to quality in technical documentation and creates a systematic organization. Diataxis identifies four modes of documentation:
```